src/components/cylc/Drawer.vue

```diff
@@ -18,164 +18,144 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <template>
   <v-navigation-drawer
     v-model="drawer"
+    id="sidebar"
```

Comment:
Use the Cylc namespace prefix "c-" for top-level component classes/ids.

```suggestion
    id="c-sidebar"
```